tests/transforms/test_sign_expand.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import numpy as np
+import pennylane as qml
+import pennylane.tape
+from pennylane import numpy as pnp
+
+"""Defines the device used for all tests"""
+
+dev = qml.device("default.qubit", wires=[0, 1, 2, 3, "Hadamard", "Target"])
+
+"""Defines circuits to be used in queueing/output tests"""
+
+with pennylane.tape.QuantumTape() as tape1:
+    qml.PauliX(0)
+    H1 = qml.Hamiltonian([1.5], [qml.PauliZ(0) @ qml.PauliZ(1)])
+    qml.expval(H1)
+
+with pennylane.tape.QuantumTape() as tape2:
+    qml.Hadamard(0)
+    qml.Hadamard(1)
+    qml.PauliZ(1)
+    qml.PauliX(2)
+    H2 = qml.Hamiltonian(
+        [1, 1],
+        [
+            qml.PauliX(0) @ qml.PauliZ(2),
+            qml.PauliZ(1),
+        ],
+    )
+    qml.expval(H2)
+
+H3 = 1.5 * qml.PauliZ(0) @ qml.PauliZ(1) + 0.3 * qml.PauliX(2)
+
+with qml.tape.QuantumTape() as tape3:
+    qml.PauliX(0)
+    qml.expval(H3)
+
+
+H4 = (
+    qml.PauliX(0) @ qml.PauliZ(2)
+    + 3 * qml.PauliZ(2)
+    - 2 * qml.PauliX(0)
+    + qml.PauliZ(2)
+    + qml.PauliZ(2)
+)
+
+with qml.tape.QuantumTape() as tape4:
+    qml.Hadamard(0)
+    qml.Hadamard(1)
+    qml.PauliZ(1)
+    qml.PauliX(2)
+
+    qml.expval(H4)
+
+TAPES = [tape1, tape2, tape3, tape4]
+OUTPUTS = [-1.5, -1, -1.5, -7]
+
+with pennylane.tape.QuantumTape() as tape1_var:
+    qml.PauliX(0)
+    H1 = qml.Hamiltonian([1.5], [qml.PauliZ(0) @ qml.PauliZ(1)])
+    qml.var(H1)
+
+with pennylane.tape.QuantumTape() as tape2_var:
+    qml.Hadamard(0)
+    qml.Hadamard(1)
+    qml.PauliZ(1)
+    qml.PauliX(2)
+    H2 = qml.Hamiltonian(
+        [1, 1],
+        [
+            qml.PauliX(0) @ qml.PauliZ(2),
+            qml.PauliZ(1),
+        ],
+    )
+    qml.var(H2)
+
+TAPES_var = [tape1_var, tape2_var]
+OUTPUTS_var = [0, 2]
+
+
+class TestSignExpand:
+    """Tests for the hamiltonian_expand transform"""
```

Comment:
Change `hamiltonian_expand` to `sign_expand` in all tests

Reply:
Changed